creating/roomscale-scaling/index.md

```diff
@@ -0,0 +1,84 @@
+keywords: [blog, creating, roomscale, blender, aframe, scale, measurement, guides]
+date_published: 2017-11-29
+date_updated: 2017-11-29
+-->
+
+# 5-Minute Guide: Scaling for Roomscale WebVR
+
+![Objects for scale!](/creating/roomscale-scaling/img/objects-for-scale.jpg)
+
+### A discussion of scaling your roomscale VR experiences for realism and surrealism
+
+<a href="demo/" class="btn btn-demo">View demo</a>
+
+<a href="https://github.com/WebVRRocks/webvrrocks/archive/master.zip" class="btn btn-source">Download source</a>
+
+---
+In this guide, I unpack the question: how do we create objects in WebVR that are scaled for the real live humans who are experiencing the site using roomscale VR?
+
+### Does the scale of the environment feel "off"?
+
+![image name](/creating/roomscale-scaling/img/surreal-scale.jpg)
+
+Stop coding for a minute and look at your WebVR scene in a roomscale (positionally tracked) headset. Does everything look huge? Is it hard to reach for things that seemed very close to the camera when viewed on the desktop monitor? The easiest reference for human scale is your own perception of how the Vive controllers intersect with other objects in the scene. Wave your hands around "through" the objects.
+
+![Sighting around objects](/creating/roomscale-scaling/img/test-by-controller-intersection.jpg)
+
+### Can I improve the user experience of the site by adjusting the scale of things?
+
+![Sighting around objects](/creating/roomscale-scaling/img/sighting-around.jpg)
+
+Be kind with your scaling. Make sure your site visitors can reach objects they need to interact with, and have a clear line of sight to objects they will need to see. Scale helps define accesibility, and also realism. Be very careful when using realistically scaled tables and chairs, for example. If the scale is believable, the user may try to lean on the table that isn't there, or sit in the virtual chair in your scene!
+
+### How large is one unit of measurement in the framework I'm using?
+
+Blender, A-Frame and Unity all use "1 meter" as their standard unit of measurement. But that does not mean that 1 meter in any of these contexts will map accurately and consistently with real world space, when viewed in VR. If you need accurate real-world scaling of your VR content, it's worth testing the scale with your system.
+
+### Using the demo models
+
+![Demo models: coffee mug and classic banana](/creating/roomscale-scaling/img/demo-viewing.jpg)
+
+In the demo scene, you'll find a few 3D models that can be used for realistic scale references. The meter stick is actually 1 meter long in real space, when viewed in Vive. In A-Frame, it is 1.026 units long. The same is true for units in Blender. Depending on the experience you are building, 2.6cm per meter might not make much difference to your users. However, if you use architectural models in your scene, tables may seem a bit too short, and hallways a bit narrower than we're accustomed to.  
+
+### DIY Measurements
```

Comment:
"DIY" may not translate well for non-native English speakers.  We may want to spell it out.